.golangci.yaml

```diff
@@ -72,6 +71,60 @@ linters-settings:
       - '.*cloudbeat.*fetching.fetchers.k8s.ProcCommonData'
       - '.*cloudbeat.*fetching.fetchers.*<anonymous>'
       - '.*cloudbeat.*flavors.benchmark.*<anonymous>'
+  revive:
+    enable-all-rules: true
+    rules:
+      - name: add-constant
+        disabled: true
+      - name: bare-return
+        disabled: true
+      - name: blank-imports
+        disabled: true
+      - name: cognitive-complexity
+        disabled: true
+      - name: confusing-naming
+        disabled: true
+      - name: confusing-results
+        disabled: true
+      - name: cyclomatic
+        disabled: true
```

Comment:
I think this is a good way to check for readability. Why are we disabling it?

Reply:
Rules like this need larger changes so we can enable them one-by-one to make reviewing easier.